VIS/Areas/VIS/Controllers/CommonController.cs

```diff
@@ -69,14 +69,14 @@ public JsonResult SaveShipment(List<Dictionary<string, string>> model, string se
         /// </summary>
         /// <param name="pref"></param>
         /// <returns></returns>
-        public JsonResult SaveInvoice(List<Dictionary<string, string>> model, string selectedItems, string C_Order_ID, string C_Invoice_ID, string M_inout_id)
+        public JsonResult SaveInvoice(List<Dictionary<string, string>> model, string selectedItems, string C_Order_ID, string C_Invoice_ID, string M_inout_id, string C_ProvisionalInvoice_ID = "0")
```

Comment:
Related js class is out in Client project, Please verify if it is working fine on that